Source/Core/WinUpdater/Platform.cpp

```diff
@@ -213,7 +219,8 @@ static bool VCRuntimeUpdate(const BuildInfo& build_info)
   CloseHandle(process_info.hProcess);
   // NOTE: Some nonzero exit codes can still be considered success (e.g. if installation was
   // bypassed because the same version already installed).
-  return has_exit_code && exit_code == EXIT_SUCCESS;
+  return has_exit_code &&
+         (exit_code == ERROR_SUCCESS || exit_code == ERROR_SUCCESS_REBOOT_REQUIRED);
```

Comment:
This feels like it should also be its own commit, since it isn't directly related to the test.

Reply:
Oh, I didn't really pay much attention to its call sites. If thats really the case, you can just ignore this one.